policy/release/test.rego

```diff
@@ -222,6 +223,40 @@ deny contains result if {
 	result := lib.result_helper(rego.metadata.chain(), [error])
 }
 
+# METADATA
+# title: Verify image disgest is present in IMAGES_PROCESSED result
+# description: >-
+#   Validates that each task with the IMAGES_PROCESSED result contains the
+#   digest of the image beind processed.
+# custom:
+#   short_name: image_digest_processed
+#   failure_msg: Test '%s' is missing '%s' from IMAGES_PROCESSED result.
```

Comment:
Perhaps
```suggestion
#   failure_msg: Task %q did not process image with digest %q.
```

Reply:
I did try this before but the result string then contains `\"` around the task name and digest. So I decided to use %s with single quotes instead. Please let me know what you think

Reply:
I think that should be okay, we already use %q [elswhere](https://github.com/enterprise-contract/ec-policies/blob/8287b80e8398b275d435089b0cfb0058bbfd054c/policy/release/sbom_spdx.rego#L78)

Reply:
so is `'%s'` : https://github.com/enterprise-contract/ec-policies/blob/main/policy/release/test.rego#L66